components/molecules/client/user/component.tsx

```diff
@@ -0,0 +1,53 @@
+import FollowBtn from '@/components/molecules/client/follow-btn';
+import Image from 'next/image';
+
+interface UserProps {
+  userId: string;
+  postId: string;
+  initialFollowed: boolean;
+  name: string;
+  followersCount: number;
+  bio: string;
+  avatarUrl: string;
+}
+
+const UserComponent: React.FC<UserProps> = ({
+  userId,
+  name,
+  followersCount,
+  bio,
+  avatarUrl,
+}) => {
+  return (
+    <div className="flex items-center justify-between w-[460px] h-[89px] gap-md p-4 border border-light-neutral-gray rounded-md">
+      <div className="flex items-center gap-md">
+        {/* Image Box with Fixed Size and Ellipse Shape */}
+        <div className="w-[50px] h-[50px] rounded-full overflow-hidden flex-shrink-0">
+          <Image
+            src={avatarUrl}
+            alt={name}
+            width={50}
+            height={50}
+            className="w-full h-full object-cover"
+          />
+        </div>
+        <div className="flex flex-col justify-center gap-sm w-[329px]">
+          <h2 className="text-[16px] font-normal leading-[25.6px] text-left text-[#2C353A]">
+            {name}
+          </h2>
+          <p className="text-[13.33px] font-normal leading-[21.33px] text-left text-[#90999E]">
+            {followersCount} followers
+          </p>
+          <p className="text-[13.33px] font-normal leading-[21.33px] text-left text-[#2C353A]">
+            {bio}
+          </p>
```

Comment:
Font styles are defined in tailwind.config.ts. use "text-<style-name>". They are also defined in figma. you can find a texts style name by checking the name under "Typography".  So replace any text styles with whatever font is specified (e.g. text-small-lg).
![image](https://github.com/user-attachments/assets/2b9a17d9-98d7-44b4-ba37-58de7f9c1ca8)

Also colors are defined in tailwind.config.ts (e.g. text-light-neutral-dark)
![image](https://github.com/user-attachments/assets/68b5ee11-3cd3-4d6f-ba58-742e97fb598f)